zerver/lib/actions.py

```diff
@@ -6451,6 +6451,17 @@ def do_remove_realm_filter(
     notify_realm_filters(realm)
 
 
+def do_update_realm_filter(realm: Realm, id: int, pattern: str, url_format_string: str) -> None:
+    pattern = pattern.strip()
+    url_format_string = url_format_string.strip()
+    realm_filter = RealmFilter.objects.get(realm=realm, pk=id)
```

Comment:
Please use `id=id)`, rather than `pk=id`; we prefer that style as it's more readable.

Reply:
It is corrected now. And I have also changed the same over [here](https://github.com/zulip/zulip/pull/17045/files#diff-9a1fd3f8a1bd0ccd686ac2d96d265d3d5dce323f6fc191adac2cb88500af6e1cR6444).